tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/check_imex_status.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+# Maximum number of attempts
+MAX_ATTEMPTS=10
+
+# Function to run the command and check status
+verify_imex_is_up() {
+    local command="/usr/bin/nvidia-imex-ctl -N -j"
+    local file_prefix="result"
+
+    for ((i=1; i<=MAX_ATTEMPTS; i++)); do
+        echo "Attempt $i for IMEX"
+
+        command_output=$(${command} 2>/dev/null)
+        # Check if the status is UP
+        if echo "$command_output" | grep -q " \"status\": \"UP\""; then
```

Comment:
What about writing/echoing the complete IMEX status (I mean the full json returned by nvidia-imex-ctl) for every iteration? In that way, in case of failures, we know what the status is and how it evolves and can facilitate troubleshooting.

Reply:
I removed this initially as I dint want the noise and i re-directed it. But I can add this as part of the code.

Reply:
done